components/commonButton.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+	<button class="button">
+		{{text}}
+	</button>
+</template>
+
+<script>
+	export default {
+		name: 'commonButton',
+		props: {
+			text: String
+		}
+	}
+</script>
+
+<style lang="scss" scoped>
+	.button {
+		background-color: $primary-red;
+		color: black;
```

Comment:
better put all colors in _variables.scss to be consistent through the whole site